### PR TITLE
[SD] Migration to latest OV tokenizers

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
- Updated README.md to contain detailed commands what to do
- Removed copy of OV tokenizers and use pip to install tokenizers via git
- Updated convert_tokenizers.py to match latest changes

### CODE REVIEW

File: image_generation/stable_diffusion_1_5/cpp/README.md [hunk 1]
```diff
@@ -1,51 +1,57 @@
-# OpenVINO Stable Diffusion (with LoRA) C++ pipeline
-The pure C++ text-to-image pipeline, driven by the OpenVINO native API for Stable Diffusion v1.5 with LMS Discrete Scheduler, supports both static and dynamic model inference. It includes advanced features like Lora integration with safetensors and OpenVINO extension for tokenizer. This demo has been tested on the Windows platform.
+# OpenVINO Stable Diffuison (with LoRA) C++ pipeline
```
Comment (Wovchena) [inline image_generation/stable_diffusion_1_5/cpp/README.md:1]: I think the spelling was correct

File: image_generation/stable_diffusion_1_5/cpp/README.md [hunk 2]
```diff
@@ -1,51 +1,57 @@
...
+The pure C++ text-to-image pipeline, driven by the OpenVINO native API for Stable Diffusion v1.5 with LMS Discrete Scheduler, supports both static and dynamic model inference. It includes advanced features like LoRA integration with safetensors and [OpenVINO extension for tokenizers](https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/custom_operations/user_ie_extensions/tokenizer/python/README.md). This demo has been tested on Windows and Linux platform.
 
-## Step 1: Prepare environment
+> [!NOTE]
+>This tutorial assumes that you are located in `<openvino.genai repo>/image_generation/stable_diffusion_1_5/cpp/` and all paths are relative to this folder.
```
Comment (Wovchena) [inline image_generation/stable_diffusion_1_5/cpp/README.md:5]:
```suggestion
>This tutorial assumes that the current working directory is `<openvino.genai repo>/image_generation/stable_diffusion_1_5/cpp/` and all paths are relative to this folder.
```

File: image_generation/stable_diffusion_1_5/cpp/README.md [hunk 3]
```diff
@@ -1,51 +1,57 @@
...
-It provides 2 ways to install `OpenVINO 2023.1.0`: 
-[conda-forge](https://anaconda.org/conda-forge/openvino) and 
-[Download archives](https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.1/windows/).
-
+SD preparation could be auto implemented with `scripts/build_dependencies.sh`. This script provides 2 ways to install `OpenVINO 2023.1.0`: [conda-forge](https://anaconda.org/conda-forge/openvino) and [Download archives](https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.1/windows/).
 ```shell
-cd scripts
-chmod +x build_dependencies.sh
-./build_dependencies.sh
+chmod +x ./scripts/build_dependencies.sh
```
Comment (Wovchena) [inline image_generation/stable_diffusion_1_5/cpp/README.md:16]: It was there before, so you can view it as out of this PR scope. Can you push `build_dependencies.sh` version with chmod +x already applied and remove this line

File: image_generation/stable_diffusion_1_5/cpp/README.md [hunk 4]
```diff
@@ -113,76 +120,59 @@ Negative prompt: (empty, here couldn't use OV tokenizer, check the issues for de
...
-rmdir /Q /S build
-mkdir build
-cd build
-cmake -G "Visual Studio 16 2019" -A x64 ^
- -DCMAKE_BUILD_TYPE=Release ^
-      ..
-cmake --build . --config Release
-```
-
+    ```shell
```
Comment (Wovchena) [inline image_generation/stable_diffusion_1_5/cpp/README.md:164]:
```suggestion
    ```bat
```

File: image_generation/stable_diffusion_1_5/cpp/README.md [hunk 5]
```diff
@@ -113,76 +120,59 @@ Negative prompt: (empty, here couldn't use OV tokenizer, check the issues for de
...
+  * OpenVINO:
+    To deployment without Conda: [Download archives* with OpenVINO](https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.1/windows/), unzip and setup environment vars with `.\setupvars.bat`
+  * Eigen:
+      ```shell
+      1. Download from https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.zip 
+      2. Unzip to path C:/Eigen3/eigen-3.4.0 
+      3. Run next step's build.bat will report error: not found Eigen3Config.cmake/eigen3-config.cmake
+      - Create build folder for Eigen and Open VS in this path C:/Eigen3/eigen-3.4.0/build
+      - Open VS's developer PS terminal to do "cmake .." and redo the CMake 
+      ```
```
Comment (Wovchena) [inline image_generation/stable_diffusion_1_5/cpp/README.md:161]:
```suggestion
      1. Download from https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.zip 
      2. Unzip to path C:/Eigen3/eigen-3.4.0 
      3. Run next step's build.bat will report error: not found Eigen3Config.cmake/eigen3-config.cmake
      - Create build folder for Eigen and Open VS in this path C:/Eigen3/eigen-3.4.0/build
      - Open VS's developer PS terminal to do "cmake .." and redo the CMake 

```

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: image_generation/stable_diffusion_1_5/cpp/README.md (hunk 1 above)
Comment (ilya-lavrenov) [reply image_generation/stable_diffusion_1_5/cpp/README.md]: fixed

File: image_generation/stable_diffusion_1_5/cpp/README.md (hunk 3 above)
Comment (ilya-lavrenov) [reply image_generation/stable_diffusion_1_5/cpp/README.md]: fixed

File: image_generation/stable_diffusion_1_5/cpp/README.md (hunk 2 above)
Comment (ilya-lavrenov) [reply image_generation/stable_diffusion_1_5/cpp/README.md]: fixed

File: image_generation/stable_diffusion_1_5/cpp/README.md (hunk 4 above)
Comment (ilya-lavrenov) [reply image_generation/stable_diffusion_1_5/cpp/README.md]: fixed

File: image_generation/stable_diffusion_1_5/cpp/README.md (hunk 5 above)
Comment (ilya-lavrenov) [reply image_generation/stable_diffusion_1_5/cpp/README.md]: fixed

**Review by Wovchena — APPROVED** [review]

**Pull request merged**